xbmc/addons/InputStream.cpp

```diff
@@ -100,6 +100,8 @@ bool CInputStream::Supports(CFileItem &fileitem)
   {
     if (addon.asString() != Name())
       return false;
+    else
+      return true;
```

Comment:
I don't like this because you can't set a break point on a particular condition.